includes/datastream.inc

```diff
@@ -219,6 +225,8 @@ function islandora_rest_get_datastream_properties(AbstractDatastream $datastream
     'controlGroup' => $datastream->controlGroup,
     'created' => (string) $datastream->createdDate,
     'versionable' => (bool) $datastream->versionable,
+    'checksum' => $datastream->checksum,
```

Comment:
To be more specific: I think we should add `checksum` and `checksumType` to any tests that `validateResponse()`s which would return those keys